hathor/event/storage/memory_storage.py

```diff
@@ -33,3 +33,15 @@ def get_event(self, key: int) -> Optional[BaseEvent]:
             return None
 
         return self.events[key]
+
+    def get_from_event(self, key: int, size: int) -> List[BaseEvent]:
+        events = []
+
+        for i in range(size):
+            current_event_key = key+i
+            if current_event_key in self.events:
```

Comment:
Why don't we use a list and just do `return self.events[key, key + size]`?

Reply:
I agree. I changed it to a list on the other PR, I'll pull the same change here.

Reply:
Also, I'd rather this be an iterator instead of a method that returns a list.

Reply:
Same. Has been changed.